### PR TITLE
[docs] Update collections.md

### PR DESCRIPTION
**Pull request by likeshumidity**
changed format to bullets and added note about the badge showing up for Questions in official collections on dashboards not in official collections

### CODE REVIEW

File: docs/exploration-and-organization/collections.md [hunk 1]
```diff
@@ -24,9 +24,15 @@ They're like file-system folders. You can put stuff in them.
...
+- These collections have a yellow badge to let people know that the items in the collection are the ones people should be looking at (or whatever "official" means to you).
+- Questions in Official collections added to Dashboards that are not in Official collections will show an Official badge next to their name on the Dashboard.
+- Questions and dashboards in Official collections are also more likely to show up at the top of search results.
+
+Pairing Official badges with [verified items](./exploration.md#verified-items) can help everyone in your Metabase sort out which questions people can trust.
+
+To add an Official badge to a collection, an admin can visit the collection and click on the dot dot dot menu (**...**) and select **Make collection official**. Admins can also remove an Official badge in the same menu. Admins can also mark a collection as Official or not when they first create the collection.
```
Comment (jeff-bruemmer) [inline docs/exploration-and-organization/collections.md:35]:
```suggestion
To add an Official badge to a collection, an admin can visit the collection and click on the dot dot dot menu (**...**) and select **Make collection official**. Admins can also remove an Official badge in the same menu. When creating a new collection, Admins also have the option of giving the collection an Official badge.
```

File: docs/exploration-and-organization/collections.md [hunk 2]
```diff
@@ -24,9 +24,15 @@ They're like file-system folders. You can put stuff in them.
 
 ![Official collections](./images/official-collection.png)
 
-Metabase admins can designate collections as "official". These collections have a yellow badge to let people know that the items in the collection are the ones people should be looking at (or whatever "official" means to you). Questions and dashboards in official collections are also more likely to show up at the top of search results. Pairing Official badges with [verified items](./exploration.md#verified-items) can help everyone in your Metabase sort out which questions people can trust.
+Metabase admins can designate collections as "official" with the following effects:
```
Comment (jeff-bruemmer) [inline docs/exploration-and-organization/collections.md:27]:
```suggestion
Metabase admins can designate collections as "official":
```

File: docs/exploration-and-organization/collections.md [hunk 3]
```diff
@@ -24,9 +24,15 @@ They're like file-system folders. You can put stuff in them.
...
 
-To add an official badge to a collection, an admin can visit the collection and click on the dot dot dot menu (**...**) and select **Make collection official**. Admins can also remove an official badge in the same menu. Admins can also mark a collection as official or not when they first create the collection.
+- These collections have a yellow badge to let people know that the items in the collection are the ones people should be looking at (or whatever "official" means to you).
```
Comment (jeff-bruemmer) [inline docs/exploration-and-organization/collections.md:29]:
```suggestion
- Official collections have a yellow badge to let people know that the items in the collection are the ones people should be looking at (or whatever "official" means to you).
```

**Review by jeff-bruemmer — APPROVED** [review]
Thanks, @likeshumidity!

**Comment by jeff-bruemmer** [comment]
@metabase-bot backport release-x.47.x

**Comment by alxnddr** [comment]
@metabase-bot backport release-x.47.x

**Comment by alxnddr** [comment]
@metabase-bot backport release-x.47.x

**Pull request merged**